src/firefly/js/ui/NaifidPanel.jsx

```diff
@@ -44,15 +44,15 @@ class NaifidPanelView extends PureComponent {
         if(searchHistory.length > 0){
             const cachedSuggList = Object.entries(searchHistory).find(([k,v])=>(v.searchVal === val));
             if(cachedSuggList){
-                return Object.entries(cachedSuggList.pop().searchRes).map( ([k,v]) => ({name:v.naifName, naifid:v.naifId}));
+                return sortBy(Object.entries(cachedSuggList.pop().searchRes).map( ([k,v]) => ({name:v.naifName, naifid:v.naifId})), 'naifid').reverse();
```

Comment:
nitpicking:  this should be written like this...
- The `if` ensure you don't get TypeError on the next line.
- Using .values instead of .entries because you don't use the key.
- Separating the array creation and the sorting makes is easier to read and debug.
```
            if (get(cachedSuggList, '0.searchRes.length') > 0) {
                const namedNaifIdAry =  Object.values(cachedSuggList.pop().searchRes).map( (v) => ({name:v.naifName, naifid:v.naifId}));
                return sortBy(namedNaifIdAry, 'naifid').reverse();
            }

```

Reply:
Fixed, thanks!!